pages/release-notes.mdx

```diff
@@ -74,6 +74,33 @@ Memgraph (v2.11.1 and lower).
 
 </Callout>
 
+## Lab v2.12.0 - Feb 28, 2024
+
+{<h3> New features and improvements </h3>}
+
+- You can change the color of nodes and edges in the graph view using
+  the color picker.
+- You can change the nodes and edges label shown in the graph view to
+  any of the following: labels, property values or none.
+- Along with OpenAI, you can now setup remote Azure OpenAI or local
+  Ollama LLM models for your GraphChat.
+- You can fine tune permissions of the GraphChat to run only read, create,
+  update and/or delete Cypher commands.
+- A quick link to GraphChat settings is now in the upper right corner
+  of GraphChat conversation.
+- Query request limit is increased from 100kB to 20MB to enable you to
+  run a lot more queries at once from query execution editor.
+
+{<h3> Bug fixes </h3>}
+
+- Query execution editor loads the previous state as expected now.
```

Comment:
Will users have the context needed to understand this? Kinda depends on how long they'd been using Lab.

I'd say: "QE editor loads the previous state when opened _again_, as opposed to the empty state bug experienced in v2.11"

Not sure it was actually an empty state, but I'd be specific.